web3.js/.flowconfig

```diff
@@ -10,6 +10,7 @@ esproposal.class_instance_fields=enable
 esproposal.class_static_fields=enable
 esproposal.decorators=ignore
 esproposal.export_star_as=enable
+esproposal.optional_chaining=enable
```

Comment:
Is this still needed? I thought I saw you mentioned that you reverted your use of optional chaining

Reply:
true, ideally I would prefer to refactor next this library to just use TypeScript to avoid issues such as esdoc

Reply:
TypeScript will be great but we are still going to need a documentation tool for describing what certain methods and arguments are for. jsdoc seems like a better alternative, there's already an issue for that here btw: https://github.com/solana-labs/solana-web3.js/issues/706

Reply:
fair, we could also do it in one change and use tsdoc

Reply:
Ah yeah, tsdoc looks even better 👍